lib/hydra/works/models/concerns/generic_file/characterization/doc.rb

```diff
@@ -0,0 +1,26 @@
+include RDF
+
+module Hydra::Works::GenericFile::Characterization
+  module Doc
+    extend ActiveSupport::Concern
+    included do
+      property :title, predicate: RDF::DC11.title
+      property :author, predicate: RDF::DC11.creator
+      property :page_count, predicate: RDF::Vocab::NFO.pageCount
+      property :file_language, predicate: RDF::DC11.language
+      property :word_count, predicate: RDF::Vocab::NFO.wordCount
+      property :character_count, predicate: RDF::Vocab::NFO.characterCount
+      property :line_count, predicate: RDF::Vocab::NFO.lineCount
+      property :character_set, predicate: RDF::URI.new("http://www.w3.org/2011/content#characterEncoding")
+      property :markup_basis, predicate: RDF::URI.new("http://www.w3.org/2011/content#doctypeName")
+      property :markup_language, predicate: RDF::URI.new("http://www.w3.org/2011/content#systemId")
+      # properties without canonical URIs
+      property :paragraph_count, predicate: RDF::URI.new("http://projecthydra.org/ns/odf/paragraphCount")
+      property :table_count, predicate: RDF::URI.new("http://projecthydra.org/ns/odf/tableCount")
+      property :graphics_count, predicate: RDF::URI.new("http://projecthydra.org/ns/odf/graphicsCount")
+
+      parser_config.merge!(file_title: :title,
```

Comment:
Updates the class configuration for mapping parsed terms to the property values.  Added a comment to explain that.